src/uphold-scripts-lint

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env sh
 
-eslint --cache . $@
+eslint --cache $@
```

Comment:
> Is this still a breaking change?

@Americas, yes because of this. Otherwise, if I update this in [notification-service](https://github.com/uphold/notification-service/blob/master/package.json#L13) without adding the dot `uphold-scripts lint .`, then it will lint 0 files 😄

Reply:
We no longer need to change the `lint` script. See recent commit.